eng/vscode/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ### Fixed
 
 - Fixed a bug in MySQL query validation logic. [[#81](https://github.com/microsoft/mcp/pull/81)]
+- Enhanced the confidence score for ALT commands [[#92](https://github.com/microsoft/mcp/pull/92)]
```

Comment:
This should say something along the lines of.. Improved description of Load Test commands... under Added section.